css/sims.css

```diff
@@ -1,3 +1,27 @@
+body {
+   background-color: #DDE8E8;
+   font-size:small;
+}
+
+.front-page-font-outline{
+   text-shadow: -0.1em 0 white, 0 0.1em white, 0.1em 0 white, 0 -0.1em white;
+}
+
+.bg-white {
```

Comment:
The text shadow? It's only for the title. If you mean the bg-white, I've changed that in a recent commit https://github.com/Zibethin/sims-website/commit/9ecdea826ad8b8c846c2f990025242c5ffe4a246